config/externalnamenottested.go

```diff
@@ -252,4 +252,75 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
 	// Batch job can be imported using the resource id
 	// /subscriptions/12345678-1234-9876-4563-123456789012/resourceGroups/resGroup1/providers/Microsoft.Batch/batchAccounts/account1/pools/pool1/jobs/job1
 	"azurerm_batch_job": config.TemplatedStringAsIdentifier("name", "/subscriptions/{{ .setup.configuration.subscription_id }}/resourceGroups/{{ .parameters.resource_group_name }}/providers/Microsoft.Batch/batchAccounts/{{ .parameters.batch_account_name }}/pools/{{ .parameters.batch_pool_name }}/jobs/{{ .externalName }}"),
+
+	// sql
+	//
+	// MS SQL Database Extended Auditing Policies can be imported using the resource id
+	// /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/group1/providers/Microsoft.Sql/servers/sqlServer1/databases/db1/extendedAuditingSettings/default
+	"azurerm_mssql_database_extended_auditing_policy": config.TemplatedStringAsIdentifier("", "/subscriptions/{{ .setup.configuration.subscription_id }}/resourceGroups/{{ .parameters.resource_group_name }}/providers/Microsoft.Sql/servers/{{ .parameters.server_id }}/databases/{{ .parameters.database_id }}/extendedAuditingSettings/"),
+	// Database Vulnerability Assessment Rule Baseline can be imported using the resource id
+	// /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/acceptanceTestResourceGroup1/providers/Microsoft.Sql/servers/mssqlserver/databases/mysqldatabase/vulnerabilityAssessments/Default/rules/VA2065/baselines/master
+	"azurerm_mssql_database_vulnerability_assessment_rule_baseline": config.TemplatedStringAsIdentifier("baseline_name", "/subscriptions/{{ .setup.configuration.subscription_id }}/resourceGroups/{{ .parameters.resource_group_name }}/providers/Microsoft.Sql/servers/{{ .parameters.server_id }}/databases/{{ .parameters.database_id }}/vulnerabilityAssessments/{{ .parameters.rule_id }}/baselines/{{ .external_name }}"),
```

Comment:
Similar to `azurerm_mssql_database_extended_auditing_policy` it seems that you use some fields that are not in schema such as `database_id` and `server_id`. Also until `Default` the value is ID of `azurerm_mssql_server_vulnerability_assessment` resource: `server_vulnerability_assessment_id`. So we can use the following one:

```suggestion
	"azurerm_mssql_database_vulnerability_assessment_rule_baseline": config.TemplatedStringAsIdentifier("baseline_name", "{{ .parameters.server_vulnerability_assessment_id }}/baselines/{{ .external_name }}"),
```

Reply:
Addressed all comments. Thank you!